generate.js

```diff
@@ -65,6 +66,18 @@ const masterfile = async function update() {
       }
       fs.writeFileSync(`./${templateName}`, JSON.stringify(newData, null, 2), 'utf8', () => { })
 
+      // compare content of both files
+      var newFile = fs.readFileSync(`./${templateName}`)
```

Comment:
```suggestion
      const newFile = fs.readFileSync(`./${templateName}`)
```
Please change these to `const`